API.md

```diff
@@ -241,8 +241,8 @@ Drag and drop SVG, HTML or Canvas using mouse or touch input.
 * [*drag*.clickDistance](https://github.com/d3/d3-drag/blob/master/README.md#drag_clickDistance) - set the click distance threshold.
 * [*drag*.on](https://github.com/d3/d3-drag/blob/master/README.md#drag_on) - listen for drag events.
 * [*event*.on](https://github.com/d3/d3-drag/blob/master/README.md#event_on) - listen for drag events on the current gesture.
-* [d3.dragDisable](https://github.com/d3/d3-drag/blob/master/README.md#dragDisable) -
-* [d3.dragEnable](https://github.com/d3/d3-drag/blob/master/README.md#dragEnable) -
+* [d3.dragDisable](https://github.com/d3/d3-drag/blob/master/README.md#dragDisable) - prevents native drag-and-drop and text selection on the specified window.
```

Comment:
I wasn't sure, so I just went with what it said in the d3-drag docs, but this makes sense. I'll make the change, thanks!